Update CHANGELOG and VERSION for v0.2.0 release (#93)
antrea-io/theia: CHANGELOG/CHANGELOG-0.2.md

# Changelog 0.2

## 0.2.0 - 2022-08-17

> ### Added

Reviewer: Some Grafana enhancement PRs may also be worth listing here. Reference: https://github.com/antrea-io/theia/issues/58

Reviewer 2: +1. There are three related PRs, can be put in the `Changed` section.

> - E2e tests for Theia functions. ([#66](https://github.com/antrea-io/theia/pull/66), [@yanjunz97]) ([#71](https://github.com/antrea-io/theia/pull/71), [@heanlan]) ([#77](https://github.com/antrea-io/theia/pull/77), [@dreamtalen])

Reviewer: As suggested by maintainers, we usually don't include tests in the changelog as they are not useful for users.

Reviewer 3: Or at least move to "Changed", as it is indeed not about new features.

  * Provide e2e tests for Theia components (ClickHouse monitor and Grafana dashboard)
  * Improve policy recommendation e2e test
- ClickHouse cluster support and data schema management. ([#55](https://github.com/antrea-io/theia/pull/55), [@yanjunz97]) ([#81](https://github.com/antrea-io/theia/pull/81), [@yanjunz97])

>   * Introduce support for Clickhouse cluster deployment with shards and replicas

Reviewer 4: ```suggestion
  * Introduce support for ClickHouse cluster deployment with shards and replicas
```
Maybe mention we replace the non-cluster mode ClickHouse by ClickHouse cluster?

>   * Introduce version control to Clickhouse data schema, which detects the data  schema version in initializing stage

Reviewer 4: ```suggestion
  * Introduce version control to ClickHouse data schema, which detects the data schema version in initializing stage
```
Do we need to mention some necessary upgrade would also be applied automatically in initializing stage?

> - Theia Command Line enhancement. ([#56](https://github.com/antrea-io/theia/pull/56), [@dreamtalen]) ([#59](https://github.com/antrea-io/theia/pull/59), [@yuntanghsu])

Reviewer 3: enhancements

How about "New Theia command line commands"

  * Commands for listing and deleting policy recommendation jobs
  * Commands to retrieve metrics in ClickHouse database

### Changed

> - Enhancement to Helm chart based deployment ([#69](https://github.com/antrea-io/theia/pull/69), [@yanjun97]) ([#73](https://github.com/antrea-io/theia/pull/73), [@antoninbas])

Reviewer 3: Enhancements

# Changelog 0.2

## 0.2.0 - 2022-08-17

### Added

- ClickHouse cluster support and data schema management. ([#55](https://github.com/antrea-io/theia/pull/55), [@yanjunz97]) ([#81](https://github.com/antrea-io/theia/pull/81), [@yanjunz97])
  * Introduce support for ClickHouse cluster deployment with shards and replicas

>   * Introduce version control to ClickHouse data schema, which detects the data schema version and applies necessay upgrade in initializing stage

Reviewer 4: ```suggestion
  * Introduce version control to ClickHouse data schema, which detects the data schema version and applies necessary upgrades in initializing stage
```

Author: Good catch, thanks!

- New Theia Command Line commands. ([#56](https://github.com/antrea-io/theia/pull/56), [@dreamtalen]) ([#59](https://github.com/antrea-io/theia/pull/59), [@yuntanghsu])
  * Commands for listing and deleting policy recommendation jobs
  * Commands to retrieve metrics in ClickHouse database

### Changed

- Enhancements to Helm chart based deployment ([#69](https://github.com/antrea-io/theia/pull/69), [@yanjun97]) ([#73](https://github.com/antrea-io/theia/pull/73), [@antoninbas])
- Move Grafana setting files to Persistent Volume ([#84](https://github.com/antrea-io/theia/pull/84), [@yanjun97])

> - E2e tests for Theia functions. ([#66](https://github.com/antrea-io/theia/pull/66), [@yanjunz97]) ([#71](https://github.com/antrea-io/theia/pull/71), [@heanlan]) ([#77](https://github.com/antrea-io/theia/pull/77), [@dreamtalen])

Reviewer 3: Move this one to be the last.

Author: Done, thanks!
